app/gateway-oss/2.3.x/upgrading.md

```diff
@@ -137,56 +135,56 @@ the database as it is migrated while the old Kong cluster keeps working until
 it is time to decommission it. For this reason, the migration is split into
 two steps, performed via commands `kong migrations up` (which does
 only non-destructive operations) and `kong migrations finish` (which puts the
-database in the final expected state for Kong 2.2.0).
+database in the final expected state for Kong 2.3.0).
 
-1. Download 2.2.0, and configure it to point to the same datastore
+1. Download 2.3.0, and configure it to point to the same datastore
    as your old (1.0 to 2.0) cluster. Run `kong migrations up`.
-2. Once that finishes running, both the old (pre-2.1) and new (2.2.0)
-   clusters can now run simultaneously. Start provisioning 2.2.0 nodes,
+2. Once that finishes running, both the old (pre-2.1) and new (2.3.0)
```

Comment:
We are not sure which version they are upgrading from at this point, but suggest the immediate version prior for best results. Maybe we should just say old and new to avoid maintenance.:

```suggestion
2. After that finishes running, both the old (2.2.0) and new (2.3.0)
```